pkg/kubelet/certificate/transport_test.go

```diff
@@ -187,7 +187,7 @@ func TestRotateShutsDownConnections(t *testing.T) {
 	}
 
 	// Check for a new cert every 10 milliseconds
-	if err := updateTransport(stop, 10*time.Millisecond, c, m, 0); err != nil {
+	if _, err := updateTransport(stop, 10*time.Millisecond, c, m, 0); err != nil {
```

Comment:
defer call `closeAllConns` in the test to increase coverage of the code change?

Reply:
I don't think so... closing stop should already call that... the point of this test is internally triggered connection closes